core/opendaq/component/include/opendaq/component_impl.h

```diff
@@ -238,14 +238,10 @@ ComponentImpl<Intf, Intfs...>::ComponentImpl(
 
     if (parent.assigned())
     {
+        this->permissionManager.setPermissions(PermissionsBuilder().inherit(true).build());
```

Comment:
I am not sure, this line is needed. I think permission manager inherits by default.

Reply:
On second thought, it is safer to leave that line here.